reinforcement_learning/bindings/cs/rl.net/ILiveModel.cs

```diff
@@ -0,0 +1,11 @@
+namespace Rl.Net
+{
+    public interface ILiveModel
+    {
+        bool TryChooseRank(string actionId, string contextJson, out RankingResponse response, ApiStatus apiStatus = null);
+        bool TryChooseRank(string actionId, string contextJson, RankingResponse response, ApiStatus apiStatus = null);
+        bool TryInit(ApiStatus apiStatus = null);
+        bool TryReportOutcome(string actionId, object outcome, ApiStatus apiStatus = null);
```

Comment:
Why not create two methods?

```csharp
        bool TryReportOutcome(string actionId, float outcome, ApiStatus apiStatus = null);
        bool TryReportOutcome(string actionId, string outcome, ApiStatus apiStatus = null);
```

Reply:
mwt client had object as the outcome type because sometimes int was also getting passed from the decision service api.
I wanted to have Jacob's opinion on what he had in mind to support that. I was conetemplating float and string myself before I came across a failed test which made me think otherwise